src/lessons/13-suspense-fundamentals.js

```diff
@@ -5,7 +5,7 @@ const cache = {}
 
 function PokemonInfo({pokemonName}) {
   const pokemon = cache[pokemonName]
-  if (!pokemon) {
+  if (!Object.hasOwnProperty.call(cache, pokemonName)) {
```

Comment:
What if we did:

```javascript
if (pokemon === undefined) {
```

A bit simpler I think.

Reply:
If it's null, then we won't hit the if block and we'll render null which is what we want. I think what I suggested is correct. Please try it.

Reply:
sorry, you are right, undefined is the initial value, it works.